SerialPrograms/Source/PokemonSV/Programs/ItemPrinter/PokemonSV_ItemPrinterRNG.h

```diff
@@ -95,14 +130,6 @@ class ItemPrinterRNG : public SingleSwitchProgramInstance, public ConfigOption::
     GoHomeWhenDoneOption GO_HOME_WHEN_DONE;
     BooleanCheckBoxOption FIX_TIME_WHEN_DONE;
 
-//    BooleanCheckBoxOption AUTO_MATERIAL_FARMING;
-//    SimpleIntegerOption<uint16_t> NUM_ROUNDS_OF_ITEM_PRINTER_TO_MATERIAL_FARM;
-    
-//    StaticTextOption MATERIAL_FARMER_DISABLED_EXPLANATION;
-    enum class MaterialFarmerTrigger{

```

Comment:
Is the new mode a new mode? Or is it replacing the existing auto-farming mode?

Reply:
Auto mode is a new mode. The existing auto-farming mode is still there. I just moved the MaterialFarmerTrigger enum class higher up in the file, for reasons I can't remember. We can move it back if you want.